### PR TITLE
refactor: control default entities display resolution

### PR DESCRIPTION
**Pull request by tschiex**
Just some minor adjustments to some sensor entities that showed a large number of digits after the decimal in the default display. There may be more sensors to "tame" than the ones I changed (among non active ones possibly).

I notice, as I create this PR, that the VERSION = "1.0.0" has been changed to VERSION = "2.1.26" and this is probably a change that should be thrown away (it likely comes from tests I did on my HA instance before submitting the PR).

Thanks to all contributors for their efforts!

### CODE REVIEW

File: custom_components/openevse/const.py [hunk 1]
```diff
@@ -46,7 +46,7 @@
 DOMAIN = "openevse"
 COORDINATOR = "coordinator"
 FW_COORDINATOR = "fw_coordinator"
-VERSION = "1.0.0"
+VERSION = "2.1.26"
```
Comment (firstof9) [inline custom_components/openevse/const.py:49]:
```suggestion
VERSION = "1.0.0"
```

This change isn't needed as the github workflow updates the version on release.

**Review by firstof9 — CHANGES_REQUESTED** [review]

**Comment by firstof9** [comment]
Don't worry about the tests, I need to fix them.

**Comment by tschiex** [comment]
I'm a newbie when it comes to PR management on GitHub. I reverted the version back to its original format; committed and pushed. It seems to be automagically inserted in the PR. Just tell me if I'm doing wrong (my apologies if so).

**Comment by firstof9** [comment]
You're doing fine, that's exactly how you do it.

**Review by firstof9 — APPROVED** [review]

**Comment by firstof9** [comment]
Thanks!

**Pull request merged**